.github/workflows/merge.yml

```diff
@@ -0,0 +1,43 @@
+name: merge
+
+on:
+  push:
+    branches: [ main ]
+
+  workflow_dispatch:
+
+concurrency: prr:deploy
+
+jobs:
+  merge:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
I think there is a v3 now.

Reply:
Should I update?

Reply:
any downsides?

Reply:
v3 just requires a higher runtime version by the looks of it https://github.com/actions/checkout#whats-new

Reply:
I'd rather keep it on v2 so we can get this in, and I'll test on v3 in parallel and we can bring it in a subsequent PR.  But happy to update too if you want.

Reply:
@JAForbes If we're going to update, let's do it from the start. Less to review and less risk all around.

Reply:
What's the status of this?